snmp/assets/configuration/spec.yaml

```diff
@@ -87,6 +87,7 @@ files:
               definition_file: f5-big-ip.yaml
             cisco-asr:
               definition_file: cisco-asr.yaml
+        hidden: true
```

Comment:
are we deprecation only the profile definition in the config (through `definition`) or also deprecating the ability to specify profiles with `definition_file` as well ?

Reply:
> are we deprecation only the profile definition in the config (through definition) or also deprecating the ability to specify profiles with definition_file as well ?

I think it's fine to deprecated both since we likely prefer users define profiles in `snmp.d/profiles` or future Profile Editor.

Reply:
what about the case where users want to configure the profile in the instance config ? I thought this needed to go through `definition_file`

Reply:
> what about the case where users want to configure the profile in the instance config ? I thought this needed to go through definition_file

@NouhaManai96 they can use `profile` in instance config even if the profiles are from `profiles` folder.